web/client/src/main.css

```
@@ -102,6 +102,24 @@
+
+  .tooltip-trigger {
+    @apply inline-flex items-center cursor-help focus:outline-none focus-visible:rounded-xs focus-visible:ring-2 focus-visible:ring-primary/30;
+  }
+
+  .tooltip-label {
+    @apply decoration-dotted underline underline-offset-4;
+  }
+
+  .floating-tooltip {
+    @apply pointer-events-none z-[200] max-w-sm rounded bg-neutral px-2 py-1 text-sm leading-tight text-neutral-content shadow-lg;
+    position: relative;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)